websites/A/Aniorb/presence.ts

```diff
@@ -138,10 +136,10 @@ presence.on("UpdateData", async () => {
     } else {
       video = document.querySelector("div > div.plyr__video-wrapper > video");
       if (video) {
-        currentTime = video.currentTime;
-        duration = video.duration;
-        paused = video.paused;
-        timestamps = presence.getTimestamps(
+        (currentTime = video.currentTime),
+          (duration = video.duration),
+          (paused = video.paused),
+          (timestamps = presence.getTimestamps(
             Math.floor(currentTime),
             Math.floor(duration)
           ));
```

Comment:
```suggestion
        currentTime = video.currentTime;
        duration = video.duration;
        paused = video.paused;
        timestamps = presence.getTimestamps(
            Math.floor(currentTime),
            Math.floor(duration)
          );
```

Reply:
there was just an extra ) at the end of timestamp.. also upload proof of it working.

Reply:
do I need to upload all screenshots again?